mods/ra/weapons/smallcaliber.yaml

```diff
@@ -32,67 +9,92 @@ FLAK-23-AA:
 	Warhead@1Dam: SpreadDamage
 		Spread: 213
 		Damage: 20
-		ValidTargets: Air, Ground, Water
+		ValidTargets: Air
 		Versus:
 			None: 40
 			Wood: 10
 			Light: 60
 			Heavy: 10
 			Concrete: 20
 		DamageTypes: Prone50Percent, TriggerProne, DefaultDeath
-	Warhead@2EffAir: CreateEffect
+	Warhead@2Eff: CreateEffect
 		Explosions: small_explosion_air
-		ValidImpactTypes: Air, AirHit
+
+ZSU-23:
+	Inherits: ^AACannon
+	Burst: 2
+	BurstDelay: 5
+	ReloadDelay: 0
+	Range: 10c0
+	Projectile: Bullet
+		Speed: 3c340
+	Warhead@1Dam: SpreadDamage
+		Damage: 12
+		Versus:
+			None: 30
+			Wood: 75
+			Light: 75
+			Heavy: 100
+			Concrete: 50
+
+FLAK-23-AA:
+	Inherits: ^AACannon
+	Warhead@1Dam: SpreadDamage
+		ValidTargets: Air, Ground, Water
 
 FLAK-23-AG:
-	ReloadDelay: 10
+	Inherits: ^AACannon
 	Range: 6c0
-	Report: aacanon3.aud
 	ValidTargets: Ground, Water
-	Projectile: Bullet
-		Speed: 1c682
-		Blockable: false
 	Warhead@1Dam: SpreadDamage
-		Spread: 213
-		Damage: 20
 		ValidTargets: Air, Ground, Water
-		Versus:
-			None: 40
-			Wood: 10
-			Light: 60
-			Heavy: 10
-			Concrete: 20
-		DamageTypes: Prone50Percent, TriggerProne, DefaultDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: small_explosion_air
```

Comment:
It was an intentional change, but not by me.

https://github.com/OpenRA/OpenRA/commit/b681f64ec4028c700aca2ee1f948730c1001d925